website/src/pages/index.tsx

```diff
@@ -26,14 +26,14 @@ import Admonition from '@theme/Admonition';
 function EAP() {
   return (
     <div>
-      <Admonition type="info" title="Join our Early Adopter Program!">
+      <Admonition type="info" title="We have participated in Hacktoberfest 2022!">
```

Comment:
Hello, shouldn't it  be 'participating' as we're still in ?

```suggestion
      <Admonition type="info" title="We are participating in Hacktoberfest 2022!">
```